com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/VariablesRequestHandler.java

```diff
@@ -331,6 +347,24 @@
         return CompletableFuture.completedFuture(response);
     }
 
+    private Types.Variable resolveLazyVariable(IDebugAdapterContext context, VariableProxy containerNode, IVariableFormatter variableFormatter,
+            Map<String, Object> options, IEvaluationProvider evaluationEngine) {
+        VariableProxy valueReferenceProxy = new VariableProxy(containerNode.getThread(), containerNode.getScope(),
+            containerNode.getProxiedVariable(), null /** container */, "");
+        valueReferenceProxy.setIndexedVariable(containerNode.isIndexedVariable());
+        valueReferenceProxy.setUnboundedType(containerNode.isUnboundedType());
+        int referenceId = context.getRecyclableIdPool().addObject(containerNode.getThreadId(), valueReferenceProxy);
+        Object proxiedVariable = containerNode.getProxiedVariable();
+        if (proxiedVariable instanceof ObjectReference) {
+            ObjectReference variable = (ObjectReference) proxiedVariable;
+            String valueString = variableFormatter.valueToString(variable, options);
+            String detailString = VariableDetailUtils.formatDetailsValue(variable, containerNode.getThread(), variableFormatter, options,
+                    evaluationEngine);
+            return new Types.Variable("", valueString + " " + detailString, "", referenceId, "");
```

Comment:
When running it on an old vscode, you will find variable name info is missing.

![image](https://user-images.githubusercontent.com/14052197/159415201-81cfcc88-0f0d-4bd4-9627-fdd58b4e739c.png)